{
  "comments": [
    {
      "key": {
        "uuid": "fd940efc_5c2b46fc",
        "filename": "extensions/phal/phal_error.cpp",
        "patchSetId": 3
      },
      "lineNbr": 515,
      "author": {
        "id": 1000004
      },
      "writtenOn": "2021-09-21T20:35:18Z",
      "side": 1,
      "message": "I must have missed this. phal has it\u0027s own trace log??",
      "revId": "62f4f3c24d92d98cb7684881da5257eeed9daf7b",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "67352dad_d98b5d57",
        "filename": "meson.build",
        "patchSetId": 3
      },
      "lineNbr": 94,
      "author": {
        "id": 1000004
      },
      "writtenOn": "2021-09-21T20:35:18Z",
      "side": 1,
      "message": "formatting issue",
      "revId": "62f4f3c24d92d98cb7684881da5257eeed9daf7b",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4",
      "unresolved": true
    }
  ]
}